lib/frontier/controller_action/edit_action.rb

```diff
@@ -1,19 +1,37 @@
 class Frontier::ControllerAction::EditAction
 
+  include Frontier::IndentRenderer
+
   attr_reader :model_configuration
 
   def initialize(model_configuration)
     @model_configuration = model_configuration
   end
 
+  # def edit
```

Comment:
Oh I get it now, maybe a comment above ?  "Outputs a method in the format of: "

Also if you lead with:
```
##
# Starts here
```

It can be an RDoc